### PR TITLE
feat: Support reset lock window password

### PR DESCRIPTION
**Pull request by yanguoyu**
Refer to https://github.com/Magickbase/neuron-public-issues/issues/394

https://github.com/nervosnetwork/neuron/assets/12881040/8199a4c3-8acc-40df-a41b-e0c93e88d049

### CODE REVIEW

**Comment by Danie0918** [comment]
@devchenyan @homura @twhy Please have a review.

File: packages/neuron-wallet/tests/services/sign-message.test.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ describe(`SignMessage`, () => {
         walletMock.mockReturnValueOnce({ isHardware: () => true })
         const signMessage = jest.fn()
         hardWalletMock.mockReturnValueOnce({ signMessage })
-        await SignMessage.sign('walletId', signInfo.address, extendedKeyInfo.password, signInfo.message)
+        await SignMessage.sign('walletId', extendedKeyInfo.password, signInfo.message, signInfo.address)
```
Comment (homura) [inline packages/neuron-wallet/tests/services/sign-message.test.ts:97]: it is easier to understand when the parameters are refactored into an object, especially when the parameter count exceeds 3

**Review by homura — APPROVED** [review]

**Review by devchenyan — APPROVED** [review]

**Comment by silySuper** [comment]
1.Tip is not same with UI document.
actual:
截屏2024-06-28 17 43 01

expect:
截屏2024-06-28 17 43 57

**Comment by yanguoyu** [comment]
> 1.UI is not same with UI document. actual: 截屏2024-06-28 17 43 01
> 
> expect: 截屏2024-06-28 17 43 57

image

I can not get this UI logic, why did the copy change from `Enter the unlock code again` to `Verify Password` when confirming the password failed? I guess these actions should be the same as changing the password on setting pages.
@Danie0918

**Comment by Danie0918** [comment]
> > 1.UI is not same with UI document. actual: 截屏2024-06-28 17 43 01
> > expect: 截屏2024-06-28 17 43 57
> 
> image
> I can not get this UI logic, why did the copy change from `Enter the unlock code again` to `Verify Password` when confirming the password failed? I guess these actions should be the same as changing the password on setting pages. @Danie0918

The prompt here is unnecessary, follow the current implementation and I'll push for UI draft changes. @silySuper

**Review by silySuper — APPROVED** [review]

**Pull request merged**